packages/fxa-auth-server/lib/routes/account.ts

```diff
@@ -643,6 +645,73 @@ export class AccountHandler {
     }
   }
 
+  async setPassword(request: AuthRequest) {
+    this.log.begin('Account.set_password', request);
+
+    const form = request.payload as any;
+    const { authPW, metricsContext } = form;
+    const { query } = request;
+    const auth = request.auth;
+    const { user: uid } = auth.credentials;
+
+    const account = await this.db.account(uid);
+    const email = account.primaryEmail.email;
```

Comment:
I thought I could get this off of `request.auth.credentials.email`, but it was `undefined` when I tested it locally -- is this the best way to get it? I only need it to pass into `this.customs.check`.

Reply:
Ben: I think it's because the access token we give from the /account/stub route doesn't give the email.